appbuild.go

```diff
@@ -158,6 +159,16 @@ func (a *AppBuild) generateBuild() *Build {
 			b.Package().DependsOn(pkgStep)
 		}
 	}
+	gopathBin := Path(Go().GetEnv("GOPATH"), "bin")
+	installVariant := a.NamedVariant("install", runtime.GOOS, runtime.GOARCH).Package(func(binaryPath, _ PathString, _, _, _ string) Task {
```

Comment:
I think it would be better to expose this as a `PackageGoInstall` command, similar to `PackageZip`, in case that's more generally useful.

Reply:
On second thought, not really. This is an `AppPackageFunc` so it only applies to `AppBuild`, and it really only applies to the install variant since it's specifically built for the current OS/arch. I'm not sure what kind of overlap could exist for this functionality.